### PR TITLE
Added new chapter/section on cluster shutdown (bsc#1191535)

### PR DESCRIPTION
**Pull request by tbazant**
Fixes https://bugzilla.suse.com/show_bug.cgi?id=1191535

rendered PDF is attached 
[admin-caasp-cluster_color_en.pdf](https://github.com/SUSE/doc-ses/files/8025175/admin-caasp-cluster_color_en.pdf)

### CODE REVIEW

File: xml/admin_caasp_cluster.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:translation>yes</dm:translation>
+   <dm:release>SES 7</dm:release>
+  </dm:docmanager>
+ </info>
+ <para>
+  This chapter introduces tasks that are performed on the whole cluster.
+ </para>
+ <sect1 xml:id="admin-caasp-cluster-shutdown">
+  <title>Shutting down and restarting the whole cluster</title>
```
Comment (asettle) [inline xml/admin_caasp_cluster.xml:23]:
```suggestion
  <title>Shutting down and restarting the cluster</title>

File: xml/admin_caasp_cluster.xml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+  <para>
+   To shut down the whole &ceph; cluster for planned maintenance tasks, follow
+   these steps:
+  </para>
+
+  <procedure>
+   <step>
+    <para>
+     Stop all clients that are using the cluster.
```
Comment (asettle) [inline xml/admin_caasp_cluster.xml:33]: @shaas could you provide the commands here? Thank you

File: xml/admin_caasp_cluster.xml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+     Set the following OSD flags:
+    </para>
+<screen>
+    &prompt.cephuser;ceph osd set noout
+    &prompt.cephuser;ceph osd set nobackfill
+    &prompt.cephuser;ceph osd set norecover
+    </screen>
+    <tip>
+     <para>
+      In addition, you can <emphasis>pause</emphasis> the cluster. When the
```
Comment (asettle) [inline xml/admin_caasp_cluster.xml:56]: Why would we pause the cluster in this procedure, @shaas ?

**Review by asettle — CHANGES_REQUESTED** [review]
Few questions inline for @shaas

File: xml/admin_caasp_cluster.xml (hunk 2 above)
Comment (shaas) [reply xml/admin_caasp_cluster.xml:33]: Well, that depends on the client which relies on the storage. I am not sure what commands you are expecting.

File: xml/admin_caasp_cluster.xml (hunk 3 above)
Comment (shaas) [reply xml/admin_caasp_cluster.xml:56]: This is list of commands which includes pause is the suggestion from the official ceph documentation. For Rook or even K8s there is no need for that.

Comment (tbazant) [reply xml/admin_caasp_cluster.xml:56]: then i'll remove pausing the cluster

File: xml/admin_caasp_cluster.xml [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+      cluster is paused, you cannot see when OSDs come back up again and no
+      &crushmap; updates will happen. To pause the cluster, set the following
+      flags:
+     </para>
+<screen>
+&prompt.cephuser;ceph osd set norebalance
+&prompt.cephuser;ceph osd set nodown
+&prompt.cephuser;ceph osd set pause
+</screen>
+    </tip>
```
Comment (tbazant) [inline xml/admin_caasp_cluster.xml:66]:
```suggestion
   
```

**Pull request merged**